RELEASE.md

```diff
@@ -0,0 +1,79 @@
+# Release Process
+
+This project uses **automated GitHub workflows** to manage versioning, releases, and publishing. Releases should **not** be performed manually.
+
+## Overview
+
+There are three main workflows involved:
+
+1. **Pull Request Checks** – validates changes before merge
+2. **Create Version Bump PR** – prepares a release by updating versions and changelog
+3. **Release on Merge to `main`** – publishes the release automatically
+
+---
+
+## 1. Pull Request Checks
+
+On every pull request, a workflow runs to ensure:
+
+- The project builds successfully
+- Code is correctly formatted and linted
+- All checks pass before merging
+
+This workflow does **not** perform any versioning or releasing.
+
+---
+
+## 2. Creating a Release (Version Bump PR)
+
+Versions must **not** be updated manually.
+
+To prepare a new release:
+
+1. Go to the **Actions** tab in GitHub
+2. Run the **“Create version bump PR”** workflow
+3. Select the release type:
+
+- `patch`
+- `minor`
+- `major`
+
+This workflow automatically:
+
+- Bumps the version in `package.json` and the lockfile
+- Updates the changelog based on changes since the previous release
+- Opens a pull request with these changes
+
+The generated pull request should be reviewed and merged like any other PR. Notice, however, that no pull request checks would be performed for it. This is a known limitation and it shouldn't be addressed.
```

Comment:
Once a chane is made to the changelog (which is expected and part of the process) CI will run regularly.

Reply:
It's true that it doesn't run CI.
But the process should be that the developer that opened the bump PR should change the changelog a bit (even just adding a new line, but usually there's some fixes needed).
Once the charge is made and committed the CI will kick in, which is expected and desired.